src/crimson/osd/pg.cc

```diff
@@ -1049,6 +1055,51 @@ PG::interruptible_future<> PG::handle_rep_op(Ref<MOSDRepOp> req)
     });
 }
 
+void PG::log_operation(
+  std::vector<pg_log_entry_t>&& logv,
+  const eversion_t &trim_to,
+  const eversion_t &roll_forward_to,
+  const eversion_t &min_last_complete_ondisk,
+  bool transaction_applied,
+  ObjectStore::Transaction &txn,
+  bool async) {
+  logger().debug("{}", __func__);
+  if (is_primary()) {
+    ceph_assert(trim_to <= peering_state.get_last_update_ondisk());
+  }
```

Comment:
I really don't want us getting into the habit of simply leaving code from classic in crimson commented out.  If you need to leave yourself a reminder for later, compose a comment with an explanation:

```
  /* TODO: when we add snap mapper and projected log support,
   * we'll likely want to update them here.
   *
   * See src/osd/PrimaryLogPG.h:log_operation for how classic
   * handles these cases.
   */
```

This way a future reader has enough context to understand what's missing without needing to fully understand classic.